shuup/front/apps/product_filter/forms.py

```diff
@@ -0,0 +1,387 @@
+                                               choices=manufacturers_choices,
+                                               required=False)
+                             )
+
+        return manufacturers
+
+    def get_basic_attributes(self, request):
+        basic_attributes = []
+        basic_attrs_settings = BasicAttributesFilterSettingsModel.objects.filter(
+            enabled=True,
+            shop=get_shop(request),
+        ).values_list(
+            'attribute_name',
+            flat=True
+        ).exclude(
+            attribute_name__in=['default_price', 'manufacturer_id', 'manufacturer']
+        )
+
+        settings_basic_attributes = Product.objects.listed(
+            shop=get_shop(request), language=get_language()
+        ).values(*basic_attrs_settings).distinct()
+
+        if settings_basic_attributes:
+            for basic_attribute in settings_basic_attributes:
+                for key, value in basic_attribute.items():
+                    if value:
+                        basic_attr_label = str(round(value))
+                        basic_attr_value = key + '__' + basic_attr_label
+                        basic_attributes.append([basic_attr_value, forms.BooleanField(
+                            widget=forms.CheckboxInput(
+                                attrs={'data-accordion-group': _(key.replace('_', ' '))}
+                            ),
+                            label=basic_attr_label,
+                            required=False)
+                        ])
+
+        return basic_attributes
+
+    def get_attributes(self, request):
+        attributes = []
+
+        attr_filters_ids = AttributesFilterSettingsModel.objects.filter(
+            shop=get_shop(request),
+            enabled=True,
+        )
+        settings_attributes = ProductAttribute.objects\
+            .select_related("attribute")\
+            .filter(untranslated_string_value__isnull=False)
+
+        if settings_attributes:
+            c = 0
+            for attribute in settings_attributes:
+                if attribute.untranslated_string_value:
+                    attr_value = str(attribute.untranslated_string_value)
+                    attr_id = str(attribute.attribute_id)
+                    attr_field_name = attr_id + '__' + attr_value
+                    attributes.append([attr_field_name, forms.BooleanField(
+                        widget=forms.CheckboxInput(
+                            attrs={'data-accordion-group': attribute.attribute.name}
+                        ),
+                        label=self.get_label(attr_value, attribute.attribute.type),
+                        required=False),
+                    ])
+                c += 1
+
+        return attributes
+
+    def get_price(self):
+        price_ranges = []
+        max_price = ShopProduct.objects.all().aggregate(Max('default_price_value'))
+
+        max_value = round(max_price['default_price_value__max'])
```

Comment:
I would guess that this fails without products. Also use shop filter above to get right products for the current shop.